workers/scripts/run_mapping_saturation_models.py

```diff
@@ -0,0 +1,127 @@
+"""Run all statistical models of the Mapping Saturation indicator and plot them."""
+
+import asyncio
+import logging
+import pickle
+from os.path import exists
+
+import matplotlib.pyplot as plt
+import numpy as np
+from dacite import from_dict
+from rpy2.rinterface_lib.embedded import RRuntimeError
+
+import ohsome_quality_analyst.geodatabase.client as db_client
+import ohsome_quality_analyst.ohsome.client as ohsome_client
+from ohsome_quality_analyst.base.indicator import LayerDefinition
+from ohsome_quality_analyst.indicators.mapping_saturation import models
+from ohsome_quality_analyst.utils.definitions import get_layer_definition
+
+
+def plot(xdata, ydata, fitted_models):
+    plt.figure()
+    prev_ax = None
+    for i, model in enumerate(fitted_models):
```

Comment:
Avoid shadowing variables from outer scope.

Reply:
This should be fixed now.

Reply:
`models` is already defined, see line 16.

Reply:
Fixed by renaming the variable.
I fixed all those issue by simply renaming the variables even though it may not be beautiful code. But this script is only used for an easy way to create all models for all regions and that only for test porpuses. So I did not want to spend much time on it.